### PR TITLE
[Doc] Chinese doc of customize_runtime

### PR DESCRIPTION
**Pull request by THU17cyz**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs_zh-CN/tutorials/customize_runtime.md [hunk 1]
```diff
@@ -1 +1,330 @@
...
+    一些模型依赖梯度裁剪技术来裁剪训练中的梯度，以稳定训练过程。举例如下：
+
+    ```python
+    optimizer_config = dict(
+        _delete_=True, grad_clip=dict(max_norm=35, norm_type=2))
+    ```
+
+    如果您的配置继承了一个已经设置了 `optimizer_config` 的基础配置，那么您可能需要 `_delete_=True` 字段来覆盖基础配置中无用的设置。详见配置文件的[说明文档](https://mmdetection.readthedocs.io/en/latest/tutorials/config.html)。
+
+- __使用动量规划器 (momentum scheduler) 来加速模型收敛__：
```
Comment (THU17cyz) [inline docs_zh-CN/tutorials/customize_runtime.md:126]: scheduler 和下面的 scheduler 有更好的翻译吗？

File: docs/tutorials/customize_runtime.md [hunk 2]
```diff
@@ -305,12 +307,12 @@ The MMCV runner will use `checkpoint_config` to initialize [`CheckpointHook`](ht
...
 ```
 
-The users could set `max_keep_ckpts` to save only small number of checkpoints or decide whether to store state dict of optimizer by `save_optimizer`. More details of the arguments are [here](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.CheckpointHook)
+The users could set `max_keep_ckpts` to save only small number of checkpoints or decide whether to store state dict of optimizer by `save_optimizer`. More details of the arguments are [here](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.CheckpointHook).
 
 #### Log config
 
 The `log_config` wraps multiple logger hooks and enables to set intervals. Now MMCV supports `WandbLoggerHook`, `MlflowLoggerHook`, and `TensorboardLoggerHook`.
-The detail usages can be found in the [doc](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.LoggerHook).
+The detail usages can be found in the [docs](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.LoggerHook).
```
Comment (Wuziyi616) [inline docs/tutorials/customize_runtime.md:315]: Shouldn't it be `detailed usage`?

File: docs_zh-CN/tutorials/customize_runtime.md [hunk 3]
```diff
@@ -1 +1,330 @@
-# 教程 5: 自定义训练配置
\ No newline at end of file
+# 教程 5：自定义运行时配置
+
+## 自定义优化器设置
+
+### 自定义 Pytorch 支持的优化器
```
Comment (Wuziyi616) [inline docs_zh-CN/tutorials/customize_runtime.md:5]: 能否将所有 `pytorch` 都统一为 `PyTorch` (`P`, `T` 大写)

File: docs_zh-CN/tutorials/customize_runtime.md (hunk 1 above)
Comment (Wuziyi616) [reply docs_zh-CN/tutorials/customize_runtime.md:126]: 我见到的翻译有 `策略` `计划表`，现在的翻译 `规划器` 我觉得也 OK？

**Review by Wuziyi616 — APPROVED** [review]
感觉问题不大

File: docs_zh-CN/tutorials/customize_runtime.md (hunk 3 above)
Comment (THU17cyz) [reply docs_zh-CN/tutorials/customize_runtime.md:5]: 把所有文档中不标准的地方都改了

File: docs_zh-CN/tutorials/customize_runtime.md [hunk 4]
```diff
@@ -1 +1,330 @@
...
+我们已经支持使用所有 PyTorch 实现的优化器，且唯一需要修改的地方就是改变配置文件中的 `optimizer` 字段。
+举个例子，如果您想使用 `ADAM` （注意到这样可能会使性能大幅下降），您可以这样修改：
+
+```python
+optimizer = dict(type='Adam', lr=0.0003, weight_decay=0.0001)
+```
+
+为了修改模型的学习率，用户只需要修改优化器配置中的 `lr` 字段。用户可以根据 PyTorch 的 [API 文档](https://pytorch.org/docs/stable/optim.html?highlight=optim#module-torch.optim) 直接设置参数。
+
+### 自定义自己实现的优化器
```
Comment (wHao-Wu) [inline docs_zh-CN/tutorials/customize_runtime.md:16]: `自定义`和`自己实现`语义好像重复了，改成`自定义优化器`就可以了

File: docs_zh-CN/tutorials/customize_runtime.md [hunk 5]
```diff
@@ -1 +1,330 @@
...
+        policy='cyclic',
+        target_ratio=(0.85 / 0.95, 1),
+        cyclic_times=1,
+        step_ratio_up=0.4,
+    )
+    ```
+
+## 自定义训练规程
+
+默认情况，我们使用阶梯式学习率衰减的 1 倍训练规程。这会调用 `MMCV` 中的 [`StepLRHook`](https://github.com/open-mmlab/mmcv/blob/v1.3.7/mmcv/runner/hooks/lr_updater.py#L167)。
```
Comment (wHao-Wu) [inline docs_zh-CN/tutorials/customize_runtime.md:149]: 要确认一下纯数字前后是否需要空格，即` 1 倍`要不要改成`1倍`

File: docs_zh-CN/tutorials/customize_runtime.md [hunk 6]
```diff
@@ -1 +1,330 @@
...
+
+**请注意**:
+
+1. 模型参数在验证期间不会被更新。
+2. 配置文件中，`runner` 里的 `max_epochs` 字段只控制训练 epoch 的数量，而不会影响验证工作流。
+3. `[('train', 1), ('val', 1)]` 和 `[('train', 1)]` 工作流不会改变 `EvalHook` 的行为，这是因为 `EvalHook` 被 `after_train_epoch` 调用，且验证工作流只会影响通过 `after_val_epoch` 调用的钩子。因此，`[('train', 1), ('val', 1)]` 和 `[('train', 1)]` 的唯一区别就是执行器 (runner) 会在每个训练 epoch 之后在验证集上计算损失。
+
+## 自定义钩子
+
+### 自定义自己实现的钩子
```
Comment (wHao-Wu) [inline docs_zh-CN/tutorials/customize_runtime.md:196]: `自定义`和`自己实现`语义重复？

File: docs_zh-CN/tutorials/customize_runtime.md [hunk 7]
```diff
@@ -1 +1,330 @@
...
+
+- 日志配置 (log_config)
+- 检查点配置 (checkpoint_config)
+- 评估 (evaluation)
+- 学习率配置 (lr_config)
+- 优化器配置 (optimizer_config)
+- 动量配置 (momentum_config)
+
+在这些钩子中，只有日志钩子拥有 `VERY_LOW` 的优先级，其他钩子的优先级均为 `NORMAL`。
+上述教程已经覆盖了如何更改 `optimizer_config`，`momentum_config`，和 `lr_config`。
```
Comment (wHao-Wu) [inline docs_zh-CN/tutorials/customize_runtime.md:296]: `覆盖`->`涉及`

File: docs_zh-CN/tutorials/customize_runtime.md (hunk 5 above)
Comment (Wuziyi616) [reply docs_zh-CN/tutorials/customize_runtime.md:149]: MMDet 有空格而 MMDet3D 合进去的文档两者皆有，要不这里先保留空格？

File: docs_zh-CN/tutorials/customize_runtime.md (hunk 6 above)
Comment (THU17cyz) [reply docs_zh-CN/tutorials/customize_runtime.md:196]: 这里自己实现是为了区分 比如 MMCV 中实现的

File: docs_zh-CN/tutorials/customize_runtime.md (hunk 4 above)
Comment (THU17cyz) [reply docs_zh-CN/tutorials/customize_runtime.md:16]: 改为了 自定义并实现优化器

File: docs_zh-CN/tutorials/customize_runtime.md (hunk 5 above)
Comment (THU17cyz) [reply docs_zh-CN/tutorials/customize_runtime.md:149]: 那我先保留着，看看泰哥和文蔚的意见

Comment (ZwwWayne) [reply docs_zh-CN/tutorials/customize_runtime.md:149]: We do not have strict rules for numbers for now. So it is fine to keep it.

**Review by ZwwWayne — APPROVED** [review]
